README.md

```diff
@@ -15,26 +17,34 @@ To generate test cases for your schema you need:
 
 - Create a parametrizer;
 - Wrap a test with `Parametrizer.parametrize` method
+- Provide a client and url of a running application instance
 
 ```python
+import pytest
+import requests
 from schemathesis import Parametrizer
 
 
 schema = Parametrizer.from_path("path/to/schema.yaml")
 
+@pytest.fixture(scope="session")
+def client():
+    return requests.Session()
```

Comment:
it would be nice to have it like this:

```
with requests.Session() as session:
    yield session
```

so the session is closed in the end